### PR TITLE
feat: Add Codespace functionality

### PR DESCRIPTION
**Pull request by uwcdc**
This PR adds Codespace functionality to the SciPy2024 devcontainer and closes issue https://github.com/uw-ssec/tutorials/issues/12

### CODE REVIEW

File: .devcontainer/Scipy2024/postBuild.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# These commands will be run after the devcontainer is built.
+
+# Setup Conda environment
+conda env create -f ~/.devcontainer/SciPy2024/environment.yml # Create environment from environment.yml
+conda init bash # Initialize conda for bash
+source ~/.bashrc # Reload bash
+conda activate scipy2024 # Activate the environment
```
Comment (lsetiawan) [inline .devcontainer/Scipy2024/postBuild.sh:7]: This is not the correct environment name

File: .devcontainer/Scipy2024/postBuild.sh [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# These commands will be run after the devcontainer is built.
+
+# Setup Conda environment
+conda env create -f ~/.devcontainer/SciPy2024/environment.yml # Create environment from environment.yml
```
Comment (lsetiawan) [inline .devcontainer/Scipy2024/postBuild.sh:4]: The pangeo base image is meant for creating another docker image from, so this configuration is not quite right. There should be a Dockerfile that uses the pangeo base image, which can read the `environment.yaml` file and create that conda environment, which then you don't really need the `postBuild.sh` script. See: https://pangeo-docker-images.readthedocs.io/en/latest/howto/custom-image.html

**Review by lsetiawan — CHANGES_REQUESTED** [review]
See my comments above.

File: .devcontainer/Scipy2024/postBuild.sh (hunk 1 above)
Comment (uwcdc) [reply .devcontainer/Scipy2024/postBuild.sh]: Sorry about that. I updated the env name.

**Review by lsetiawan — CHANGES_REQUESTED** [review]
The comment in https://github.com/uw-ssec/tutorials/pull/14/files#r1543522585 still need to be resolved. Thanks!

**Comment by lsetiawan** [comment]
superseded by #18

**Pull request closed without merging**